api/src/paths/project/{projectId}/attachments/list.ts

```diff
@@ -102,12 +126,14 @@ export function getAttachments(): RequestHandler {
     const projectId = Number(req.params.projectId);
     const connection = getDBConnection(req['keycloak_token']);
 
+    const queryObject: fileType = req.query || {};
+
     try {
       await connection.open();
 
       const attachmentService = new AttachmentService(connection);
 
-      const data = await attachmentService.getAttachments(projectId);
+      const data = await attachmentService.getAttachmentsByType(projectId, queryObject.type);
```

Comment:
Got a few comments about who we are parsing/passing type around.  Will bug you in discord about it tomrorow.

Reply:
Completed